src/broadcast/participant.rs

```diff
@@ -13,14 +13,23 @@ use crate::{
     participant::{ProcessOutcome, ProtocolParticipant},
     protocol::ParticipantIdentifier,
     run_only_once_per_tag,
-    storage::{PersistentStorageType, Storage},
+    storage::{Storable, Storage},
     Identifier,
 };
 use rand::{CryptoRng, RngCore};
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 use tracing::{info, instrument};
 
+// Storage identifiers for the broadcast protocol.
+#[derive(Clone, Copy, Debug, Serialize)]
+#[serde(tag = "Broadcast")]
+enum StorageType {
+    Votes,
+}
```

Comment:
question: You don't have to change this, but I'm curious. Does this have to be an enum, or could we just do a unit struct? I don't anticipate adding extra storables to broadcast, and from the type bounds in `storage.rs`, it looks like it just needs to implement `Storable`.

Reply:
Probably not, I just followed the existing framework used in the other protocols.

Given that this is going away soon due to #181 I'll leave it as is.